### ISSUE TITLE
how to set up email?

### ISSUE THREAD

**Issue by nickzengxl** [solidnerd/docker-bookstack#70]
when I enable register feature and use email to validate.
But when I register a new a account, it's failed. Because bookstack can send out email.

**Comment by nickzengxl** [comment]
This is my docker-compose file:

```
version: '2'
services:
  mysql:
    image: mysql:5.7.21
    environment:
    - MYSQL_ROOT_PASSWORD=secret
    - MYSQL_DATABASE=bookstack
    - MYSQL_USER=bookstack
    - MYSQL_PASSWORD=secret
    volumes:
    - /srv/bookstack/mysql-data:/var/lib/mysql

  bookstack:
    image: solidnerd/bookstack:0.21.0
    depends_on:
    - mysql
    environment:
    - DB_HOST=mysql:3306
    - DB_DATABASE=bookstack
    - DB_USERNAME=bookstack
    - DB_PASSWORD=secret
    - MAIL_HOST=smtp.exmail.qq.com
    - MAIL_USERNAME=book-haha@test.com
    - MAIL_PASSWORD=xxxxx
    - MAIL_PORT=465
    - MAIL_FROM=book-haha@test.com
    - MAIL_FROM_NAME=book-haha
    volumes:
    - /srv/bookstack/uploads:/var/www/bookstack/public/uploads
    - /srv/bookstack/storage-uploads:/var/www/bookstack/public/storage
    ports:
    - "8080:80"
```

**Comment by iFloris** [comment]
@NickZengXL I am having the same problem. Did you manage to make any progress on this?

**Comment by iFloris** [comment]
@NickZengXL this now seems to have been solved (at least for me) as can be seen in #22
